src/index.html

```diff
@@ -3,18 +3,25 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="
+          width=device-width, 
+          user-scalable=no, 
+          initial-scale=1.0, 
+          maximum-scale=1.0, 
+          minimum-scale=1.0
+        "
+    >
```

Comment:
![image](https://user-images.githubusercontent.com/121725856/215112333-018b88b6-0501-4004-8bcf-541f7b82ea48.png)

Fix like it shows at the example